taskt/UI/Forms/frmScriptEngine.cs

```diff
@@ -44,15 +44,22 @@ public partial class frmScriptEngine : ThemedForm
         private bool _closeWhenDone = false;
         public string Result { get; set; }
         public bool IsNewTaskSteppedInto { get; set; }
+        public bool IsNewTaskResumed { get; set; }
+        public bool IsNewTaskCancelled { get; set; }
+        public bool IsHiddenTaskEngine { get; set; }
+        public int DebugLineNumber { get; set; }
+        public bool IsDebugMode { get; set; }
         #endregion
```

Comment:
I can definitely see where these flags can start to get clunky/hard to track. Should take special care when making engine documentation.